apps/prairielearn/src/server.js

```diff
@@ -847,6 +860,18 @@ export async function initExpress() {
     '/pl/course/:course_id(\\d+)/elements',
     (await import('./pages/elementFiles/elementFiles.js')).default,
   );
+  app.use(
+    '/pl/course_instance/:course_instance_id(\\d+)/sharedElements/course/:producing_course_id(\\d+)/elements',
+    (await import('./pages/elementFiles/elementFiles.js')).default,
+  );
+  app.use(
+    '/pl/course_instance/:course_instance_id(\\d+)/sharedElements/course/:producing_course_id(\\d+)/instructor/elements',
+    (await import('./pages/elementFiles/elementFiles.js')).default,
+  );
+  app.use(
+    '/pl/course/:course_id(\\d+)/sharedElements/course/:producing_course_id(\\d+)/elements',
+    (await import('./pages/elementFiles/elementFiles.js')).default,
+  );
```

Comment:
Review the comments above this block; these routes exist solely for backwards-compatibility. We don't need to add them for brand-new routes.

Reply:
Is the comment actually correct? 

I thought these routes are also always being used in local dev as well.

Reply:
🙈 I independently discovered the same thing while working on #9864. Yes, let's change the comment to more accurately reflect why these are here!

Reply:
Would you like to do the comment in that PR? That would seem to make more sense than doing it here, since that PR is specifically about fixing the situation.